### PR TITLE
feat: add coinbase to ccxt fetcher

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

File: src/fetchers/ccxt/CcxtFetcher.ts [hunk 1]
```diff
@@ -56,7 +56,9 @@ export class CcxtFetcher extends BaseFetcher {
 
     for (const ticker of Object.values(response) as Ticker[]) {
       const pairSymbol = ticker.symbol;
-      const lastPrice = ticker.last as number;
+      const lastPrice = this.exchange.name === "Coinbase"
+        ? ticker.info
```
Comment (hatskier) [inline src/fetchers/ccxt/CcxtFetcher.ts:60]: Hmm, looks a bit hacky. Can you please describe why the Coinbase fetcher needs another logic than other ccxt fetchers?

**Review by hatskier — CHANGES_REQUESTED** [review]
Why do you update a lot of other manifests (not related to coinbase) here? If this PR also handles the data cleaning, can you please rename it? :)

Can you also update the list of sources metadata according to [this document](https://github.com/redstone-finance/redstone-node/blob/main/docs/ADD_NEW_SOURCE.md)?

**Comment by cehali** [comment]
I run scripts for ccxt and this is result 😄 Yes, I will remove all other changes and leave ones regarding coinbase

File: src/fetchers/ccxt/CcxtFetcher.ts (hunk 1 above)
Comment (cehali) [reply src/fetchers/ccxt/CcxtFetcher.ts:60]: Here example: 
image

**Comment by cehali** [comment]
Removed plain coinbase from ccxt and deleted remaining of coinbase fetcher

**Pull request merged**